packages/smooth_app/lib/pages/product/new_product_page.dart

```diff
@@ -209,6 +212,48 @@ class _ProductPageState extends State<ProductPage> with TraceableClientMixin {
     );
   }
 
+  Widget _buildWebsiteWidget(String website) => InkWell(
+        onTap: () async {
+          if (!website.startsWith('http')) {
```

Comment:
Https sites will break this logic

Reply:
What can we do? Either hope that all https site redirect from http, or that all sites are https.
I chose the former. Are there stats about it?

Reply:
Yeah I think we can just leave this like that. I was more thinking about 

https://xyz.com 

Getting into http://https://xyz.com

Reply:
We're safe: `https` does start with `http`.